docs/CHANGELOG.asciidoc

```diff
@@ -47,6 +47,8 @@
 * Improve robustness of anomaly detection to bad input data. (See {ml-pull}1114[#1114].)
 * Adds new `num_matches` and `preferred_to_categories` fields to category output.
   (See {ml-pull}1062[#1062])
+* Ensure classification and regression results agree for all our target operating systems.
+  (See {ml-pull}1127[1127].)
```

Comment:
The `1127` inside the square brackets should be preceded by `#`.

Reply:
See [5f6500b](https://github.com/elastic/ml-cpp/pull/1127/commits/5f6500b278ec5c3b95dc83a25c51e9fb71231618).